src/pages/Login.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { Button, Card, Space } from 'antd';
+import { GoogleCircleFilled } from '@ant-design/icons';
+import { auth, googleProvider } from '../services/FirebaseService';
+
+import './Login.css';
+
+function Login() {
+    const googleSignIn = e => {
```

Comment:
Also another good alternative that is mostly used nowadays when interacting with promises, is the `async/await` syntax. That is also something that is good to pick up. [Async/Await](https://javascript.info/async-await)

The code below can be simplified like this if using async/await

```
const googleSignIn = async () => {
   try {
      const response = await auth.signInWithPopup(googleProvider)
      const user = response.user
     
      console.log(response.user)
   } catch (error){
      console.error(error.message)
   }
} 
```

Reply:
You can also omit the `e` parameter since it's not being used. Also another reminder, when naming variables, don't use single letters, let's be as descriptive as possible.